VariablesHomework/VariablesHomework.cs

```diff
@@ -60,86 +61,86 @@ public void Operators()
         {
             int a = 0;
             int b = a + 13; 
-            Assert.IsTrue(b == /*Answer here*/); // Make this pass
+            Assert.IsTrue(b == 13); // Make this pass
 
             float f = 12.3f;
             float c = f - 7.43f;
-            Assert.IsTrue(c == /*Answer here*/); // Make this pass
-            Assert.IsTrue(f == /*Answer here*/); // Make this pass
+            Assert.IsTrue(c == 4.87); // Make this pass
```

Comment:
We'll need to talk about this one in class. It doesn't pass the test (even if you add the correct suffix), and we need to talk about why.